h2o-docs/src/product/security.rst

```diff
@@ -144,6 +144,27 @@ What is being Secured Today
 3. Internal H2O node-to-H2O node communication can be encrypted.
 
 
+Enforcing System-Level Command-Line Arguments in h2odriver.jar
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+System administrators can create a configuration file with implicit arguments of h2odriver and use it to make sure the H2O cluster is started with the specified security settings. 
+
+1. Create the config file in **/etc/h2o/h2odriver.args**.
+2. Specify the default command-line options that you want to enforce. Note that each argument must be on a separate line. For example:
+
+ ::
+
+   h2o_ssl_jks_internal=keystore.jks
+   h2o_ssl_jks_password=password
+   h2o_ssl_jts_internal=truststore.jks
+   h2o_ssl_jts_password=password
+
+3. Use the ``-Dai.h2o.args.config`` option to specify the configuration file when starting H2O.
```

Comment:
This configuration is just to change the default name of the file. The user does not need to specify it if the file `**/etc/h2o/h2odriver.args**` exists

Reply:
So can I just remove step 3?

Reply:
I would keep it, but just write there that `h2odriver.args` is default and people don't have to specify this if they named the file as this, just in cases they used custom name

Reply:
I would remove it, this is an internal parameter.

On the other hand we should include the description of this functionality in SW as well (with a proper name of the file).